docs/pages/router/reference/static-rendering.mdx

```diff
@@ -283,6 +283,51 @@ export default function Page() {
 }
 ```
 
+## Fonts
+
+Expo Font has automatic static optimization for font loading in Expo Router. This means when you load a font with `expo-font`, Expo CLI will automatically extract the font resource and embed it in the page's HTML, enabling preloading, faster hydration, and reduced layout shift.
+
+The following block will load Inter into the namespace and statically optimize on web:
+
+```js app/home.js
+import { Text } from 'react-native';
+import { useFonts } from 'expo-font';
+
+export default function App() {
+  /* @info Expo CLI automatically finds and extracts this font during compilation. */
+  const [isLoaded, error] = useFonts({
```

Comment:
```suggestion
  const [isLoaded] = useFonts({
```

Not sure but why are we referencing `error` here if we are not using it? Can `error` be used in the `if` condition on line 304? For example:

```
  if (!isLoaded && !error) {
    return null;
  }
```